uber/decorators.py

```diff
@@ -485,3 +485,20 @@ def __call__(self, func):
         redirect_func = create_redirect(self.url or '../' + get_module_name(func) + '/' + func.__name__)
         setattr(root, self.alias_name or func.__name__, redirect_func)
         return func
+
+
+def id_required(func):
+    @wraps(func)
+    def check_id(self, **params):
+        message = "ID NOT FOUND"
+        session = params['session']
+        if params.get('id'):
+            try:
+                uuid.UUID(params['id'])
+            except ValueError:
+                message = "UUID is an invalid hexadecimal string."
+            else:
+                if session.query(sa.Attendee).filter(sa.Attendee.id == params['id']).first():
+                    return func(**params)
```

Comment:
You should set a different message for if their ID exists but they couldn't be found in the database. Something like "You weren't found in our database," like Eli suggested above.